modules/managing-namespace-auto-pruning-policies.adoc

```diff
@@ -331,14 +295,14 @@ $ curl -X POST -H "Authorization: Bearer <access_token>" -H "Content-Type: appli
 +
 [source,terminal]
 ----
-$ curl -X GET -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/<user>/autoprunepolicy/8c03f995-ca6f-4928-b98d-d75ed8c14859
+$ curl -X GET -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/user/autoprunepolicy/8c03f995-ca6f-4928-b98d-d75ed8c14859
 ----
 +
 Alternatively, you can include the UUID:
 +
 [source,terminal]
 ----
-$ curl -X GET -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/<user>/autoprunepolicy/
+$ curl -X GET -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/user/autoprunepolicy/
```

Comment:
If we write `Alternatively, you can include the UUID:` here,  the line 298 and line 305  should swap places, right?

Reply:
Fixed. Nice catch! I just appended the UUID to line 305.